ocs_ci/ocs/platform_nodes.py

```diff
@@ -230,13 +236,15 @@ def start_nodes(self, nodes, wait=True):
         )
         self.vsphere.start_vms(vms)
 
-    def restart_nodes(self, nodes, force=True):
+    def restart_nodes(self, nodes, force=False, wait=True):
```

Comment:
this changes functionality.
Are we sure this will not impact any existing operation?

Reply:
*. In the current implementation, since we are setting the force arg to True by default it stops the VM by direct poweroff.
*. With direct VMware reboot, reboot can be done in two ways,
1)Hard reboot(VM Reset)
 2) Soft reboot(Guest Reboot)
I thought soft reboot is most common scenario compared to VM reset so set False by default, this anyways can be changed according to the test case.

I ran existing testcases which are using this function and didn't see any failure and I am also planning to run multiple tiers/workloads to make sure nothing is breaking because of this change.

Please let me know if I am missing anything here and you think setting to false breaks existing operations here.